scripts/irods/test/test_resource_types.py

```diff
@@ -532,6 +532,31 @@ def acPostProcForParallelTransferReceived(rule_args, callback, rei):
         except:
             print('Skipping for plugin name ['+self.plugin_name+']')
 
+    def test_unixfilesystem_freespace_with_istream__issue_7160(self):
```

Comment:
Can we add a couple more words to the test name to describe what it's about?

It's not clear to me what the goal of the test is.

It seems the test is proving `istream` works when the freespace options are enabled on a resource. Is that the goal of the test?

Reply:
What about...
1. `test_istream_works_with_freespace_and_minimum_free_space_for_create_in_bytes_configured__issue_7160`
2. `test_istream_does_not_fail_with_freespace_and_minimum_free_space_for_create_in_bytes_configured__issue_7160`
3. `test_istream_works_with_minimum_free_space_for_create_in_bytes_configured__issue_7160`
4. `test_istream_does_not_fail_with_freespace_and_minimum_free_space_for_create_in_bytes_configured__issue_7160`

Reply:
(1) and (3) work the best to me. They are both clear IMO.

Thoughts?

Reply:
I vote 3. Thanks!